analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/MetadataReferenceFacade.cs

```diff
@@ -184,6 +184,13 @@ internal static class MetadataReferenceFacade
             NuGetMetadataReference.SystemSecurityPermissions();
 #endif
 
+        internal static References SystemThreading =>
+#if NETFRAMEWORK
+            FrameworkMetadataReference.SystemThreading;
```

Comment:
This is not needed under .NET Fx
```suggestion
            Enumerable.Empty<MetadataReference>();
```